ui/package.json

```diff
@@ -34,6 +37,7 @@
     "vue-i18n": "^8.18.2",
     "vue-property-decorator": "^8.4.2",
     "vue-router": "^3.3.4",
+    "vue2-editor": "^2.10.2",
```

Comment:
Do we need both `vue2-editor` and `tiptap`?

Reply:
No, we only need tiptap, I will remove it in a second